rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -615,15 +661,45 @@ pub fn tags_to_lanes(
 ) -> Result<RoadFromTags, RoadError> {
     let mut warnings = RoadWarnings::default();
 
+    // Early return if we find unimplemented tags.
     unsupported(tags, locale, &mut warnings)?;
 
-    let mut road: RoadBuilder = RoadBuilder::from(tags, locale, &mut warnings)?;
+    // Parse lane count schemas first.
+    let oneway =
+        Oneway::from(tags.is_any("oneway", &["yes", "-1"]) || tags.is("junction", "roundabout"));
+    let busway = BuswayScheme::new(tags, locale, oneway, &mut warnings);
+    // let bus_lanes = BusLanes::from(tags, locale, &oneway, &mut warnings);
+    // TEMP: lets use the lanes:bus schema to summarise bus lanes for the lanes scheme.
+    let lanes_bus = LanesBusScheme {
+        forward: busway
+            .forward_side_direction
+            .map_or(0, |o| if o.is_some() { 1 } else { 0 }),
+        backward: busway
+            .backward_side_direction
+            .map_or(0, |o| if o.is_some() { 1 } else { 0 }),
+        bothways: Infer::Default(0),
+    };
+    let centre_turn_lanes = CentreTurnLaneScheme::new(tags, oneway, locale, &mut warnings);
+    let lanes = LanesScheme::new(
+        tags,
+        /*highway,*/ oneway,
```

Comment:
Same here -- bits of commented code checked in can add up quickly and get overwhelming. (I'm going to pick on another project I've worked on, https://github.com/Urban-Analytics/RAMP-UA/blob/c5ecee2ab8eebd2b9cffc41956c7c83195b709d1/coding/initialise/population_initialisation.py#L386)

Reply:
Sensible, I'll do a pass, and get some TODOs too.